### DISCUSSION TITLE
Cannot create an app in ArgoCD from a repository that is connected with my username and a token (PAT)

### DISCUSSION THREAD

**Discussion by aychakarma** [argoproj/argo-cd#18481] [General]
0
Edit summary: The issue has nothing to do with Azure DevOps. Remove Azure DevOps tag.

I have OpenShift GitOps 1.12 configured on an OpenShift cluster v4.15, and I am experiencing a problem with creating applications in ArgoCD.

I can successfully connect my private repository using my username and a personal access token (PAT). However, when I try to create an app, I encounter the following error:

Error 128 authentication failed git fetch origin --rpc error: code = Unknown desc = failed to initialize repository resources: rpc error: code = Internal desc = Failed to fetch default: git fetch origin --tags --force --prune failed exit status 128: fatal: Authentication failed

Does anyone have a solution for this issue?

Thank you for your assistance!

What I tried:

Connecting with my username and password, but it didn't work.
Including the token in the URL (https://token@), but when I did this, it said "cannot read password."

**Comment by naveenb30** [comment]
@aychakarma : I'm also having same exact issue, did you find any solution for this ?
